common/app/common/configuration.scala

```diff
@@ -242,6 +242,10 @@ class GuardianConfiguration extends Logging {
     lazy val jsLocation = configuration.getStringProperty("googletag.js.location").getOrElse("//www.googletagservices.com/tag/js/gpt.js")
   }
 
+  object apstag {
```

Comment:
Sorry for the long PR ! I should have broken it into smaller chunks. You are right `apstag` is not obvious is a9 related, although a developer working for a9 should know about the `apstag` because that's how the amazon library is been called attaching a screenshot as you need login to access their docs. As this is to put the parameter that holds the apstag API key for AWS Parameter stored I though apstag should match it. Also apstag.js is a new file with the library itself

![Screenshot 2019-12-11 at 15 57 48](https://user-images.githubusercontent.com/51630004/70637391-0e77e300-1c2f-11ea-9292-000e80cd8345.png)

Reply:
Totally worth having a comment there. my TLA brain cells are running out :)

Reply:
renamed & commented 🙂